ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -438,6 +443,12 @@ public ConditionsPage clickOnConditionsWidgetLink() {
         return new ConditionsPage(this);
     }
     
+    public void deletePatient(String REASON) {
+    	clickOn(DELETE_PATIENT);
+    	setTextToFieldNoEnter(DELETE_PATIENT_REASON,REASON);
+    	clickOn(DELETE_PATIENT_CONFIRM);		
```

Comment:
@sherrif10 some thing worth looking into is the indention of `deletePatient()` function implementation. Take a look into into it before geeks do the magic of merging. Take care @dkayiwa @ibacher

Reply:
I think the everything is fine with identation,anything to improve, deletePatient() , and **deletePatient ()** , the latter is smarter than the last one

Reply:
A proper code indention would make the first letter of the functions at line `447` `448` `449 `to be in a vertical line with letter `i` of the word `public` present at line code 446. But of now they are in a vertical line with letter `b` of the word `public` which should not be that way based on the existing code indention in respect with the epic.